.docs/user-guide/usage.md

```diff
@@ -0,0 +1,210 @@
+# Usage
+Report status, Initiate storage operations
+
+---
+
+## Summary
+General format:
+
+```sh
+$ rexray [option flags] [command]
+```
+
+Get general or command specific help
+
+```sh
+$ rexray --help
+```
+
+```sh
+$ rexray [command] --help
+```
+
+## Global Option Flags
+These can be invoked with all commands
+
+| Flag | Alternate form | Description |
+| --- | --- | --- |
+| `-c` | `--config=""` | Path to a custom REX-Ray configuration file |
+| `-h` | `--host=""` | designate libStorage host to execute command |
+| `-l` | `--logLevel=""` | error, warn, info, debug |
+| `-s` | `--service=""` | designate specific libStorage service command pertains to |
+| `-v` | `--verbose=true` | emit verbose information to console |
+
+## Commands
+| Command | Description |
+| --- | --- |
+| version | return REX-Ray and libStorage versions |
+| env | return effective current configuration, in form normalized as environment variables |
+| adapter | return list of configured services, or list of host instances manageable by a service |
+| device | list or operate (mount,unmount,format) on OS devices |
+| service | manage REX-Ray running as an OS service |
+| volume | list or operate (create,remove,attach,detach,mount,unmount) on volumes |
+| help | return list of commands and global option flags |
+
+## Examples
```

Comment:
Hi @cantbewong,

Please ensure there is text beneath headings instead of just tabular information or list data. All headings should be followed by some short, descriptive piece of information about that section's content.